fetch.bs

```diff
@@ -110,6 +110,12 @@ urlPrefix:https://tc39.es/ecma262/#;type:dfn;spec:ecma-262
         "href": "https://datatracker.ietf.org/doc/html/draft-ietf-dnsop-svcb-https",
         "publisher": "IETF",
         "title": "Service binding and parameter specification via the DNS (DNS SVCB and HTTPS RRs)"
+    },
+    "WEBDRIVER-BIDI": {
+        "authors": [],
+        "href": "https://w3c.github.io/webdriver-bidi/",
+        "publisher": "W3C",
+        "title": "WebDriver BiDi"
```

Comment:
Any reason this isn't indexed automatically? Normally this shouldn't be needed.

Reply:
Observationally:
```
FATAL ERROR: Couldn't find 'WEBDRIVER-BIDI' in bibliography data. Did you mean:
  webdriver2
  webdriver
  webdriver1
  webmidi
  entries-api
```

I don't think any of those suggestions are what we want. Maybe something has to happen on the WebDriver side to get this into the biblo data, but I'm not sure what.

Reply:
filed https://github.com/tobie/specref/pull/746 to add it to specref (it would have been added automatically if it had been published as TR doc)